packages/button/src/mwc-button.ts

```diff
@@ -64,7 +64,7 @@ export class Button extends LitElement {
           ?disabled="${this.disabled}"
           aria-label="${this.label || this.icon}">
         ${this.icon ? html`<span class="material-icons mdc-button__icon">${this.icon}</span>` : ''}
-        ${this.label}
+        ${html`<span class="mdc-button__label">${this.label}</span>`}
```

Comment:
Why do you need to nest templates here? Wouldn't just this work the same
```
<span class="mdc-button__label">${this.label}</span>
```

Reply:
Thanks @elf-pavlik, fixed. You'll see that I was also wrong with the head branch, so I had to open a new pull request, sorry.